applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,130 @@
+
+Or, if the controlling chip is the nRF52840 of the nRF9160 DK, the following files must also be included:
+
+* :file:`overlay-zephyr-modem-nrf9160dk-nrf52840.conf` - To define the power pin.
+* :file:`overlay-zephyr-modem-nrf9160dk-nrf52840.overlay` - To configure the UART to be routed between the nRF9160 and the nRF52840 of the DK.
+
+Finally, if you want more verbose logging that includes the AT commands and responses, you can enable debug logging by uncommenting ``CONFIG_SLM_LOG_LEVEL_DBG=y`` in :file:`prj.conf`.
+
+Controlling chip running Zephyr
+===============================
+
+Configuration files found in Zephyr's :zephyr:code-sample:`cellular-modem` sample are a good starting point.
+
+Specifically, regardless of what the controlling chip is, the Kconfig options found in the following files:
+
+* :file:`prj.conf` - This file enables various Zephyr APIs, most of which are needed for proper functioning of the application.
+* :file:`boards/nrf9160dk_nrf52840.conf` - This file tailors the configuration of the modem subsystem and driver to the SLM.
+  It makes the application's logs be output on UART 0 and also enables the debug logs of the cellular modem driver.
+  If you do not want the debug logs output by the driver, you may turn them off by removing ``CONFIG_MODEM_LOG_LEVEL_DBG=y``.
+
+In addition, depending on what the controlling chip is, the following devicetree overlay files.
+They define the modem along with the UART it is connected to and its power pin.
+
+If the controlling chip is an external MCU:
+
+* :file:`boards/nrf9160dk_nrf9160_ns.overlay` - The UART configuration and power pin can be customized according to your setup.
+
+If the controlling chip is the nRF52840 of the nRF9160 DK:
+
+* :file:`boards/nrf9160dk_nrf52840.overlay` - The UART and power pin are configured to be routed to the nRF9160.
+
+Developing the Zephyr application
+*********************************
+
+To get started developing the Zephyr application running on the controlling chip, look at the code of Zephyr's :zephyr:code-sample:`cellular-modem` sample to see how the modem is managed and used.
+You can even compile, flash and run the sample to verify proper operation of the modem.
+
+Flashing and running
+********************
+
+When built with the Zephyr-compatible modem configuration, SLM will put the nRF91 Series SiP to deep sleep when powered on.
+Zephyr's cellular modem driver running on the controlling chip will take care of waking up the nRF91 Series SiP, so it is advised to first flash SLM to the nRF91 Series SiP.
+
+However, before flashing the SLM built with the Zephyr-compatible modem configuration, make sure that the nRF91 Series modem has been set to the desired system mode.
+Do this by issuing an ``AT%XSYSTEMMODE`` command followed by an ``AT+CFUN=0`` command so that the modem saves the system mode to NVM.
```

Comment:
```suggestion
Do this by issuing an ``AT%XSYSTEMMODE`` command followed by an ``AT+CFUN=0`` command so that the modem saves the system mode to NVM.
You will need a regular SLM running in nrRF91 SiP, to run AT-commands.
```